pkg/reconciler/contour/config/contour.go

```diff
@@ -36,14 +36,18 @@ const (
 	visibilityConfigKey = "visibility"
 	// nolint:gosec // Not an actual secret.
 	defaultTLSSecretConfigKey = "default-tls-secret"
+	timeoutPolicyIdleKey      = "timeoutPolicy-idle"
+	timeoutPolicyResponseKey  = "timeoutPolicy-response"
```

Comment:
it is weird that this mixes styles, I'd use `timeout-policy-idle` 🙏

Reply:
Done